lib/libmspack/mspack/cabd.c

```diff
@@ -396,15 +398,17 @@ static int cabd_read_headers(struct mspack_system *sys,
   if (cab->base.flags & cfheadPREV_CABINET) {
     cab->base.prevname = cabd_read_string(sys, fh, &err);
     if (err) return err;
-    cab->base.previnfo = cabd_read_string(sys, fh, &err);
+    // in at least one CAB the nextname was set but nextinfo was empty; so we allow an empty nextinfo here
```

Comment:
NOTE: This is the core fix here on the mspack side. Some of these multipart cabs seem to have empty nextinfo's and we need to safely ignore that